bin/openshift-include.sh

```diff
@@ -4,6 +4,73 @@
 # This script is not intended to be run directly
 # Assumes bin/common.sh has been sourced
 
+function ocVersionCheck {
+  origIFS=$IFS
+  IFS=$'\n'
+
+  allArr=($(oc version 2>/dev/null))
+  IFS=$origIFS
+
+  for (( i=0; i<${#allArr[@]}; i++ )); do
+    # Split the line into an array
+    verArr=(${allArr[$i]})
+    if [ ${#verArr[@]} -eq 3 ]; then
+      verType="${verArr[0]}"
+      ver="${verArr[2]}"
+      if [ "$verType" == "Client" ]; then
+          ver="${verArr[2]}"
+          if [[ $ver =~ v(([0-9]+)\.([0-9]+)\.([0-9]+)) ]]; then
+            OC_FULL_VERSION=${BASH_REMATCH[1]}
+            OC_MAJOR_VERSION=${BASH_REMATCH[2]}
+            OC_MINOR_VERSION=${BASH_REMATCH[3]}
+            OC_PATCH_VERSION=${BASH_REMATCH[4]}
+          else
+            echo "Unable to parse client version: [$ver]"
+          fi
+      elif [ "$verType" == "Server" ]; then
+          ver="${verArr[2]}"
+          if [[ $ver =~ (([0-9]+)\.([0-9]+)\.([0-9]+)) ]]; then
+            OSHIFT_FULL_VERSION=${BASH_REMATCH[1]}
+            OSHIFT_MAJOR_VERSION=${BASH_REMATCH[2]}
+            OSHIFT_MINOR_VERSION=${BASH_REMATCH[3]}
+            OSHIFT_PATCH_VERSION=${BASH_REMATCH[4]}
+          else
```

Comment:
Looks like an opportunity for consolidating code or creating a semver parse function....but we may never need that again. Probably not worth the effort.

Reply:
I think a refactor is in the cards for the future